templates/solutions/ai-infrastructure/index.html

```diff
@@ -0,0 +1,454 @@
+{% extends 'base_index.html' %}
+
+{% block title %}Full-stack AI infrastructure{% endblock %}
+
+{% block meta_description %}
+  Fast-track your AI journey with a proven and supported open source
+  AI infrastructure stack. Start quickly in the public cloud, then run
+  your models at scale in the data centre and at the edge.
+{% endblock meta_description %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1oYV0LsLNg2yUn6Y81AMMX-9jDnojUT8H8kLqqizOZ-M/edit
+{% endblock meta_copydoc %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+  <section class="p-strip is-shallow u-no-padding--bottom">
+    <div class="p-section">
+      <div class="row">
+        <div class="col-start-large-4 col-9">
+          <div class="p-section--shallow">
+            <h1 class="u-no-margin--bottom">Your full stack for AI infrastructure at scale</h1>
+          </div>
+          <div class="p-section--shallow">
+            <p>
+              We make AI infrastructure easy so you can focus on your models. Get all the infrastructure components you need for AI, from the operating system to the MLOps platform and all the way to the secure edge.
+            </p>
+          </div>
+          <hr class="u-hide--small u-hide--medium" />
+          <a class="p-button--positive"
+             href="/contact-us"
+             aria-controls="ai-infrastructure-modal">Contact Canonical</a>
+        </div>
+      </div>
+      <div class="p-suru"></div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Why Canonical for enterprise AI infrastructure?</h2>
+      </div>
+      <div class="col">
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">Stable and supported end-to-end stack optimised for AI performance</li>
+          <li class="p-list__item is-ticked">Control your TCO with predictable pricing per node</li>
+          <li class="p-list__item is-ticked">Design and deploy your AI infrastructure with expert guidance</li>
+          <li class="p-list__item is-ticked">Fast-track compliance with security across all layers of the stack</li>
+        </ul>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="p-section--shallow">
+      <div class="row--50-50">
+        <hr class="p-rule" />
+        <div class="col">
+          <h2>Deploy on any platform, scale with a hybrid cloud strategy</h2>
+        </div>
+        <div class="col">
+          <p>
+            Choose the ideal AI infrastructure for your use cases – for instance, start quickly with no risk and low investment on public clouds, then move workloads to your own data centre as you scale.
```

Comment:
We usually wrap dashes that have spaces on either side
```suggestion
            Choose the ideal AI infrastructure for your use cases &mdash; for instance, start quickly with no risk and low investment on public clouds, then move workloads to your own data centre as you scale.
```